src/Setup/Bundle/PackageGroups.wxs

```diff
@@ -64,6 +64,12 @@
                 <MsiProperty Name='INSTALLFOLDER' Value='[InstallFolder]' />
             </MsiPackage>
             <?endif?>
+
+            <?ifdef VS2017Available ?>
```

Comment:
MSBuild property != WiX variable, I expected it at https://github.com/wixtoolset/wix3/blob/develop/tools/WixBuild.wixproj.props. The libs didn't get installed on my machine.